tools/SourceKit/tools/sourcekitd/lib/API/CMakeLists.txt

```diff
@@ -16,5 +16,8 @@ endif()
 
 add_sourcekit_library(sourcekitdAPI
   ${sourcekitdAPI_sources}
-  DEPENDS SourceKitSupport
+  DEPENDS
+      SourceKitSupport SourceKitSwiftLang
+      # Clang dependencies, required by SourceKitSwiftLang
+      clangFormat clangToolingCore clangDriver
```

Comment:
Yes, I wondered that as well. I was surprised I had to do this, to be honest. Maybe @llvm-beanz would know?